src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -205,14 +229,16 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
             latestTokenPrice.value = value.result["price"];
           }
         } else if (value.action === "UPDATE") {
+          console.log(value.result.symbol);
```

Comment:
can we get rid of that? overall cleanin up logs after development is a good practice imo